CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Note-It
+Contributing to this repo? Follow the steps below to set-up the project in your local system, making changes and making a pull request.
+
+## How to contribute (overall process)
+
+1. Fork the project, using the gray `Fork` button in the top right of this page
+2. Make any changes in your forked repo
+3. On this repo, click `Pull Requests` (which is the third option at the top of this page after the options `Code` and `Issues`) and raise a Pull Request by clicking the green `New Pull Request` button and selecting your fork from the right drop down field.
+
+Questions can be asked by raising an `Issue`.
+
+## How to clone repo and make changes locally after forking
+
+- Click on the green `Code` button, then either the HTTPS or SSH option and click the icon to copy the URL. This will give you a copy of the project, so you can play around with it locally on your computer.
+
+- Using Git on your local machine and paste in the URL. Do this to download the forked copy of this repo to your computer.
+
+```
+  git clone https://github.com/yourGithubUsername/Note-It.git
+```
+
+- Switch to the cloned folder. This can be done with Gitbash or the integrated terminal in the VSCode editor.
```

Comment:
Don't mention about VS code, this project will not work there. Just mention about git bash